SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/WorkerCoordinator/SpatialConfig/cloud_launch_sim_player_deployment.json

```diff
@@ -0,0 +1,40 @@
+{
+  "template": "sim_players",
+  "world": {
+    "chunkEdgeLengthMeters": 50,
+    "snapshots": {
+      "snapshotWritePeriodSeconds": 0
+    },
+    "dimensions": {
+      "xMeters": 300,
+      "zMeters": 300
+    }
+  },
+  "load_balancing": {
+    "layer_configurations": [
+      {
+        "layer": "SimulatedPlayerCoordinator",
+        "rectangle_grid": {
+          "cols": 10,
+          "rows": 1
```

Comment:
Why this configuration?

Reply:
These are placeholder values and will be overwritten by the deployment launcher. `cols` specifies the number of coordinator workers (and therefore simulated players) to start.